src/backend/InvenTree/InvenTree/schema.py

```diff
@@ -105,9 +105,11 @@ def get_operation(
             parameters = operation.get('parameters', [])
             for parameter in parameters:
                 if parameter['name'] == 'ordering':
-                    parameter['description'] = (
-                        f'{parameter["description"]} Possible fields: {", ".join(ordering_fields)}.'
-                    )
+                    schema_order = []
+                    for field in ordering_fields:
+                        schema_order.append(field)
+                        schema_order.append('-' + field)
```

Comment:
Adding to the list twice gives duplicate entries.

```suggestion
```

Reply:
Ah, I've used that but seeing it as an enum value looked out of place. My mistake.